src/middleware/cors.ts

```diff
@@ -2,7 +2,6 @@ import { NextFunction, Request, Response } from "express";
 
 export function corsMiddleware(req: Request, res: Response, next: NextFunction): void {
     res.header("Access-Control-Allow-Origin", "*");
-    res.header("Access-Control-Allow-Headers", "X-Requested-With, Content-Type, Accept");
```

Comment:
Will this break cors when developing without nginx?

Reply:
tests seemed to pass - Content-Type and Accept are whitelisted headers and X-Requested-With shouldn't be used.

I can write tests to check for PREFLIGHT

Reply:
you're right content-type has to be explicitly whitelisted